src/chrome/content/rules/Ubisoft-Entertainment.xml

```diff
@@ -62,32 +65,42 @@
 	<target host="static15.ubi.com" />
 	<target host="static17.ubi.com" />
 	<target host="static19.ubi.com" />
+	<target host="store.ubi.com" />
 	<target host="support.ubi.com" />
 	<target host="tools.ubi.com" />
 	<target host="ubibar.ubi.com" />
 	<target host="uplay.ubi.com" />
+	<target host="www.uplay.ubi.com" />
 	<target host="uts.ubi.com" />
-	<target host="legal.ubi.com" />
 
 	<securecookie host=".*\.ubi\.com$" name=".+" />
 
 	<!--
-		static*.cdn.ubi.com
+		static*.cdn:
 			- Handled by http→https rule: static(2|5|7).cdn
-			- Handled by akamaihd.net rule: static(3|7|9|10|11|14|17|19).cdn
+			- Handled by akamaihd.net rule: static(|3|7|9|10|11|14|17|19).cdn
 			- In problematic subdomains: static13.cdn
 	-->
-	<rule from="^http://static(3|7|9|10|11|14|17|19)\.cdn\.ubi\.com/"
+	<rule from="^http://static(|3|7|9|10|11|14|17|19)\.cdn\.ubi\.com/"
```

Comment:
This regex looks wrong. Did you mean `(3|7|9|10|11|14|17|19)?` ?

Reply:
No, I needed to include `static.cdn.ubi.com` so I added the extra `|`.

Reply:
My bad. Not too familiar with regex yet. Fixing!